components/micro-gateway-core/src/main/ballerina/src/gateway/utils/utils.bal

```diff
@@ -760,7 +760,6 @@ public function initAuthHandlers() {
     //Initializes apikey handler
     jwt:JwtValidatorConfig apiKeyValidatorConfig = {
         issuer: getConfigValue(API_KEY_INSTANCE_ID, ISSUER, DEFAULT_API_KEY_ISSUER),
-        audience: getConfigValue(API_KEY_INSTANCE_ID, AUDIENCE, DEFAULT_AUDIENCE),
```

Comment:
So the fix for #1104 if removing audience validation for any apiKey (APIM or MGW Key)? 
Anyway I'd be extremely happy if you sent these two fixes in two different commits.

Reply:
Yes, since we removed aud from validation, no point in issuing an API Key with aud claim. Added as 2 commits as you requested.